privacyidea/models.py

```diff
@@ -2474,3 +2476,204 @@ def __init__(self, username, realm, resolver, authentication,
         self.authentication = authentication
         self.first_auth = first_auth
         self.last_auth = last_auth
+
+### Periodic Tasks
+
+class PeriodicTask(MethodsMixin, db.Model):
+    """
+    This class stores tasks that should be run periodically.
+    """
+    __tablename__ = 'periodictask'
+    id = db.Column(db.Integer, Sequence("periodictask_seq"), primary_key=True)
+    name = db.Column(db.Unicode(), unique=True, nullable=False)
+    active = db.Column(db.Boolean, default=True, nullable=False)
+    interval = db.Column(db.Interval, nullable=False)
```

Comment:
Ah, so should we allow a scheduling possibility similar to crontab? i.e. with minute, hour, day of month, month, day of week? If yes, we should discuss if we want to store this in several columns, or if we want to store it as a Unicode with a well-defined format.

Reply:
Honestly, I am not sure if we want to be that complicated like the crontab format. Well, maybe we want to, since some know it.
But honestly I thinks something like:

* A certain time of day   (11 pm)
* A certain day of the week  (Saturday)
* A certain date of the month  (28th day in month)
* all X minutes...

Makes sense. And basically we are at the cron format. Well, we do not need to allow something like "*/4". But bottomline: I would make this a ``unicode`` column and have the application apply the logic.

Reply:
OK, sounds good!